synapse/tests/test_cortex.py

```diff
@@ -29,6 +29,52 @@ class CortexTest(s_t_utils.SynTest):
     '''
     The tests that should be run with different types of layers
     '''
+    async def test_cortex_usernotifs(self):
+
+        async def testUserNotifs(core):
+            async with core.getLocalProxy() as proxy:
+                root = core.auth.rootuser.iden
+                indx = await proxy.addUserNotif(root, 'hehe', {'foo': 'bar'})
+                self.nn(indx)
+                item = await proxy.getUserNotif(indx)
+                self.eq(root, item[0])
+                self.eq('hehe', item[2])
+                self.eq({'foo': 'bar'}, item[3])
+                msgs = [x async for x in proxy.iterUserNotifs(root)]
+
+                self.len(1, msgs)
+                self.eq(root, msgs[0][1][0])
+                self.eq('hehe', msgs[0][1][2])
+                self.eq({'foo': 'bar'}, msgs[0][1][3])
+
+                await proxy.delUserNotif(indx)
+                self.none(await proxy.getUserNotif(indx))
+
+                retn = []
+                done = asyncio.Event()
+                async def watcher():
+                    async for item in proxy.watchAllUserNotifs():
+                        retn.append(item)
+                        done.set()
+                        return
+                core.schedCoro(watcher())
+                await asyncio.sleep(0.1)
```

Comment:
if this is just to get the watcher() routine running we can just call sleep(0).

Reply:
Not necessarily. In the case where it's double telepath it could short circuit the reply